packages/@o3r/analytics/src/services/event-track/event-track.service.ts

```diff
@@ -44,12 +44,12 @@ import {
 } from './event-track.configuration';
 
 /** The initial value of the performance measurements */
-export const performanceMarksInitialState: PerfEventPayload = {
+export const performanceMarksInitialState: Readonly<PerfEventPayload> = {
```

Comment:
Why do we put `Readonly` on some but not on all of the `as const` ?

Reply:
Not sure I got the question :S.
`readonly<...>` is mandatory when the constant is declared `as const` but, as it is exposed, should also have an explicit type following an interface.
When there are not exported, it does not need to have the type defined but only to `satisfy` it. So no need `Readonly<...>`.